first-react-app/src/components/homeComponents/ModalCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+function ModalCard(props: {
```

Comment:
you should define a named interface - `ModalCardProps` and export it instead of copy-pasting in `first-react-app/src/components/homeComponents/ModalCard.test.tsx` file.

```suggestion
const ModalCard = (props: ModalCardProps) => {
```